stack/python/stack.py

```diff
@@ -0,0 +1,52 @@
+class Stack:
+    """
+    Class for operations related to stack.
+    """
+    def __init__(self):
+        """
+        Constructor.
+        """
+        self.stack = []    # Initialise empty list
+
+    def push(self, dataval):
+        """
+        Method to push items in stack.
+
+        dataval: Value that user needs to push.
+        """
+        self.stack.append(dataval)    # Add dataval at the end of the list.
+
+    def pop(self):
+        """
+        Method for removing the last element from the stack.
+
+        Returns
+        -------
+        element that is popped.
+        """
+        if len(self.stack) <= 0:
+            return ("No element in the Stack")    # Raise an error is if stack is empty.
+        else:
+            return self.stack.pop()
+
+    def top(self):
+        return self.stack[len(self.stack)-1]    # Show the last element of the list.
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmplnn5ba1l/stack/python/stack.py
+++ b/tmp/tmplnn5ba1l/stack/python/stack.py
@@ -30,7 +30,8 @@
             return self.stack.pop()
 
     def top(self):
-        return self.stack[len(self.stack)-1]    # Show the last element of the list.
+        # Show the last element of the list.
+        return self.stack[len(self.stack)-1]
 
     def display(self):
         print(*self.stack)
```